source/core/source/forge.js

```diff
@@ -1763,27 +1770,30 @@ var gca_forge = {
 			};
 
 			// Prefix
-			for (let i = 0; i < result.id.prefix.length; i++) {
-				let id = result.id.prefix[i];
+			for(const [id, prefix] of Object.entries(gca_data_recipesNames.prefixes)) {
 				this.info.prefix[id] = {
-					id : id,
-					level : result.level.prefix[i],
-					name : result.name.prefix[id]
+					id,
+					level : gca_data_recipes.getRecipe(id, null, null).prefix.level,
+					name : prefix[gca_locale._getLang()] ?? prefix['us'],
```

Comment:
Avoid calling get functions inside for loops when they are not affected by the iterations of the loop.
Just add:
```
const lang = gca_locale._getLang();
```
Outside the loop, and use that instead.